images/fcos-bastion-image/root/usr/bin/reserve-hosts.sh

```diff
@@ -119,6 +119,7 @@ printf '%s\n' "${OUTPUT[@]}" >> "$RESERVED_FILE"
 if [ x"$REQUEST_VIPS" == x"true" ]; then
     FIRST_HOST_ID=$(echo "${CANDIDATES_HOSTS[0]}" | cut -f3 -d,)
     echo "{\"ingress_vip\": \"${INGRESS_VIPS_SUBNET_PREFIX:-192.168.82.}${FIRST_HOST_ID}\", \"api_vip\": \"${API_VIPS_SUBNET_PREFIX:-192.168.81.}${FIRST_HOST_ID}\"}" > "${BUILD_DIR}/vips.json"
+    echo "{\"ingress_vip_v6\": \"${IPV6_INGRESS_VIPS_SUBNET_PREFIX:-fd99:2222:3456::2:}${FIRST_HOST_ID}\", \"api_vip_v6\": \"${IPV6_API_VIPS_SUBNET_PREFIX:-fd99:2222:3456::1:}${FIRST_HOST_ID}\"}" >> "${BUILD_DIR}/vips.json"
```

Comment:
FIRST_HOST_ID can be either 1, 2, or 3 characters. Can we print it as an hexadecimal representation and force it to be `00$FIRST_HOST_ID_HEX`?

Reply:
Also, you should add the two keys in the row above as you are printing two json maps into a plain json file... which is an invalid json

Reply:
yes, make sense, they should combined on one row
regardless, there is a lot of shellcheck failures not related to my changes

Reply:
Done

Reply:
shellcheck is complaining about:
Avoid x-prefix in comparisons as it no longer serves a purpose.

should I remove these ?

Reply:
Shellcheck job passed after removing x-prefix and unused var, but images still failing

Reply:
All checks passed